.stylelintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "stylelint-config-standard-scss"
+    ],
+    "plugins": [
+        "stylelint-selector-bem-pattern"
+    ],
+    "ignoreFiles": ["sass/tna-toolkit/**/*.scss"],
+    "rules": {
+        "at-rule-empty-line-before": null,
+        "block-no-empty": null,
+        "declaration-empty-line-before": null,
+        "property-no-vendor-prefix": null,
+        "value-no-vendor-prefix": null,
+        "value-keyword-case": null,
+        "scss/dollar-variable-empty-line-before": null,
+        "scss/double-slash-comment-empty-line-before": null,
+        "selector-class-pattern": null,
+        "plugin/selector-bem-pattern": {
+            "preset": "bem"
+        },
+        "declaration-block-no-duplicate-properties": {
+            "ignore": ["consecutive-duplicates-with-different-values"]
+        },
+        "scss/at-extend-no-missing-placeholder": null,
```

Comment:
I added this because there seems to be a widely used pattern in the codebase where instead of using `class="alert alert--warning"` we just have `class="alert--warning"` - and the BEM modifier class `@extends` the block class.

Reply:
Sure thing. I would like to eventually remove this because some styles have not strictly adhered enough to the BEM strategy but if we need it for now then this is fine. 👍🏼